src/schemas/space.json

```diff
@@ -234,8 +234,31 @@
           "required": ["name"],
           "additionalProperties": false
         },
-        "delegation": {
-          "type": "boolean"
+        "delegationPortal": {
+          "standard": {
```

Comment:
I would call this "delegationType" similar than on SX-UI. The type may not be a real "standard".